.github/actions/private-tangle/setup/action.yml

```
@@ -0,0 +1,23 @@
+name: 'private-tangle-setup'
+description: 'Setup a private tangle'
+runs:
+  using: "composite"
+  steps:
+    - name: Setup private tangle
+      shell: bash
+      run: |
+        git clone https://github.com/iotaledger/hornet.git
```

Quick win: Cloning just 14 MB instead of 174 MB.

```suggestion
        git clone --depth=1 https://github.com/iotaledger/hornet.git
```